### DISCUSSION TITLE
Update to PHP 8.4 impacting image

### DISCUSSION THREAD

**Discussion by syntax922** [YOURLS/YOURLS#3826] [General]
Has anyone else had issues with the new image from Nov 21? I'm not sure why it isn't version locked, but the change seems to be breaking. I noticed in the Aura/sql group (which is an upstream package) that they noticed a breaking change and are having to make changes to their libraries. 

I'm by no means an expert on images, but I was able to make it work by rolling back the php version change and rebuilding my own image.

**Comment by LeoColomb** [comment]
See https://github.com/YOURLS/containers/issues/311
